test/server-test.js

```diff
@@ -26,16 +28,61 @@ describe('Server', function(){
   describe('GET /', function(){
     it('should return a 200', function(done){
       this.request.get('/', function(error, response){
-        if (error) { done(error); }
+        if (error) { done(error) }
         assert.equal(response.statusCode, 200);
         done();
       });
     });
   });
 
-  describe('GET /api/secrets/:id', function(){
+  describe('GET /api/foods/:id', function(){
+    beforeEach(function(){
+      app.locals.foods = {
+        wowowow: 'I am a banana'
+      }
+    });
+
+    it('should return a 404 if the resource is not found', function(done){
+      this.request.get('/api/foods/bahaha', function(error, response){
+        if(error) { done(error) }
+        assert.equal(response.statusCode, 404);
+        done();
+      });
+    });
+
+    it('should return a 200 if the resource is found', function(done){
+      this.request.get('/api/foods/wowowow', function(error, response){
+        if(error) { done(error) }
+        assert.equal(response.statusCode, 200);
+        assert(response.body.includes('wowowow'), 'ID was not included');
+        assert(response.body.includes('I am a banana'), 'ID was not included');
+          done();
+      });
+    });
+  });
+
+  describe('POST /api/foods', function(){
     beforeEach(function(){
-      app.locals.secrets = {}
-    })
-  })
+      app.locals.foods = {}
+    });
+
+    it('should not return 404', function(done){
+      this.request.post('/api/foods', function(error, response){
+        if (error) { done(error) }
+        assert.notEqual(response.statusCode, 404)
+        done();
+      });
+    });
+
+    it('should recieve and store data', function(done){
```

Comment:
@case-eee @neight-allen we cannot get this test to pass. When we console log in, we see that our message is not being saved. it is still an empty object. ``` app.locals.foods is {} ```